components/bl-list-component/src/list.js

```diff
@@ -0,0 +1,16 @@
+import { ListContent } from './list-content'
+
+export function List ({ component, eventHandlers, children, as=component.type }) {
```

Comment:
`<List as={ component.type } component={ component } eventHandlers={ eventHandlers }/>`

As i see we don't pass children here.

But in another place:

```
<List
          component={ component }
          children={ item.children }
          eventHandlers={ eventHandlers }
        />
```

Could we add an props 'items' - and only this props can accept collection of objects to render?